README.md

```diff
@@ -5,65 +5,35 @@ A React component library to power all Palmetto UI.
 
 In the project directory, you can run:
 
-### `yarn start`
+### `yarn storybook`
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Launches a local storybook instance to test components in a sandbox.<br />
+Open [http://localhost:9009](http://localhost:9009) to view it in the browser.
```

Comment:
haha how do we choose dev ports at palmetto?

Reply:
@ericyliu these are defaults from storybook config.